scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -304,11 +304,18 @@
 
                                       :cook-expected-state/killed
                                       (case pod-synthesized-state-modified
-                                        ; This is interesting. This indicates that something deleted it behind our back!
-                                        ; Weird. We always update datomic first. Could happen if someone manually removed stuff from kubernetes.
+                                        ; TODO: This can also legitimately occur in a
+                                        ; normal occurrence if someone submits a kill request on a job that fails to launch.
                                         :missing (do
-                                                   (log-weird-state compute-cluster pod-name
-                                                                    cook-expected-state-dict k8s-actual-state-dict)
+                                                   (log/warn "In compute cluster" name ", pod" pod-name "in a weird cook expected state:" cook-expected-state-dict "and k8s actual state" k8s-actual-state-dict)
```

Comment:
Why aren't we using `log-weird-state`?

Reply:
This case is slightly strange, but expected under some race conditions. :( 
Its too useful to ignore; its found several bugs, but can't be an error.

Reply:
Ah, got it. If it's expected, then shouldn't it be `INFO` level? Seems like things that we expect to happen on a regular basis should be `INFO`. 

Maybe we should create a `log-state` function for cases like this where it's expected but we don't want to ignore it.

Reply:
As discussed, we'll do it if we have a second instance of downgraded warnings. For now, keep it this way.